### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by Qoosim**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+<body>
+  <div class="wrapper">
+    <header>
+      <span class="line-span"></span>
+      <img src="image/logo.png" alt="youtube logo">
+    
+      <input type="text" placeholder="Search" >
+      <!-- <button class="search"> -->
+        <!-- <img src="image/search-icon.png" alt="Search-icon" title="Search" id="mirror"> -->
+        <a href="#"><span class="search-mirror"></span></a>
```
Comment (samgaco) [inline index.html]: The search icon should be enclosed within `<button>` tags

**Comment by samgaco** [comment]
### Hi @Qoosim and @mihrab34 :hand: 

### Good work so far :+1: the website is looking like the original one but there are some points to address:

* The **search icon** should be enclosed in button tags

* The **navigation bar** should have at least three embedded images on the rightmost part of the navbar that represents icons and/or an avatar
icons7

* The videos and the thumbnails should show the **number of views**.

* Read about [HTML semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) elements and use them. You could use `<main>` tag for the dominant content of the body or `<footer>` for the footer.

* The description under the video is overflowing the subscribe button and the profile of the uploader. There are also some grey margins on both sides. You could try to correct these issues.
Screenshot_2019-08-21_19-44-21

### Please :pray: work on these points and submit back for another code review (remember to send for review the link to the pull request. Regards! :muscle:

**Review by samgaco — CHANGES_REQUESTED** [review]
### Hi @Qoosim and @mihrab34 :hand: 

### Good work so far :+1: the website is looking like the original one but there are some points to address:

* The **search icon** should be enclosed in button tags

* The **navigation bar** should have at least three embedded images on the rightmost part of the navbar that represents icons and/or an avatar
icons7

* The videos and the thumbnails should show the **number of views**.

* Read about [HTML semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) elements and use them. You could use `<main>` tag for the dominant content of the body or `<footer>` for the footer.

* The description under the video is overflowing the subscribe button and the profile of the uploader. There are also some grey margins on both sides. You could try to correct these issues.
Screenshot_2019-08-21_19-44-21

### Please :pray: work on these points and submit back for another code review (remember to send for review the link to the pull request. Regards! :muscle:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+</head>
+<body>
+  <div class="wrapper">
+    <header>
+      <span class="line-span"></span>
+      <img src="image/logo.png" alt="youtube logo">
+    
+      <input type="text" placeholder="Search" >
+      <!-- <button class="search"> -->
+        <!-- <img src="image/search-icon.png" alt="Search-icon" title="Search" id="mirror"> -->
```
Comment (samgaco) [inline index.html]: It's better not to have commented code :smiley:

**Review by samgaco — CHANGES_REQUESTED** [review]
And remember to get rid of the commented code in your project.

**Review by uche-inyama — CHANGES_REQUESTED** [review]
Hi team, :hand: you did a good job,:handshake: You have  implemented the changes as requested. 

However, I want to make a suggestion, when you want to add icons next time, use [font awesome](https://fontawesome.com/) check it out here. 

However, there is still one more edit to make, your **README.md file** 

**First**, remove this:  :point_down: 
#### <<<<<<< HEAD This is our first collaborative project in Microverse.
#### This is our first collaborative project in Microverse  ff616738828fe544affa50214900e7fb184c3761  

 from your **README** 

Also, make your **README** more descriptive, by adding:
  - [ ] Live version of your project, use [GitHack](https://raw.githack.com/)
  - [ ] Write on how a vistor to your github account can clone the project.
  - [ ] For more detailed information for vistors, provide a link to Odin project.
  - [ ] The collaborators of the project, adding the links to your individual github accounts or any site that promotes your skill as a software developer.

 Update your README and submit another code review form.

#### Guys, you are almost there. :muscle:

**Pull request merged**